Specs/Core/FeatureDetectionSpec.js

```diff
@@ -115,4 +115,12 @@ defineSuite([
             expect(FeatureDetection.imageRenderingValue()).not.toBeDefined();
         }
     });
+
+    it('detects WebP support', function(done) {
+        FeatureDetection.supportsWebp()
+            .then(function(supportsWebp) {
+                expect(typeof supportsWebp).toEqual('boolean');
+                done();
```

Comment:
Remove `done` and just `return FeatureDetection.supportsWebp()`

Reply:
Like this?

```
it('detects WebP support', function() {
        return FeatureDetection.supportsWebp()
    });
```

This doesn't actually verify that the returned value is a boolean.